presto-main/src/main/java/com/facebook/presto/sql/planner/QueryPlanner.java

```diff
@@ -416,10 +416,11 @@ private PlanBuilder aggregate(PlanBuilder subPlan, QuerySpecification node)
             groupingSetsSymbolsBuilder.add(groupingColumns.build());
         }
 
+        List<List<Symbol>> groupingSetsSymbols = groupingSetsSymbolsBuilder.build();
```

Comment:
The analysis already contains a variable called groupingSets with a different type